Hannah-Sten/TeXiFy-IDEA
# Texlive package install

#### Summary of additions and changes

* When using tlmgr to install a package, update tlmgr first if needed.
* Cancel the installing of a package when tlmgr cannot find it, display error balloon.

#### How to test this pull request

* Use the inspection action to install a fake package. See that an error balloon is displayed, and that the install action is available afterwards.
